src/main/java/com/celements/rights/access/DefaultRightsAccessFacade.java

```diff
@@ -79,4 +79,16 @@ public boolean hasAccessLevel(EntityReference ref, EAccessLevel level, XWikiUser
         user, ret);
     return ret;
   }
+
+  @Override
+  public boolean isLoggedIn() {
+    boolean ret = false;
+    if (!XWikiRightService.GUEST_USER_FULLNAME.equals(getContext().getUser())) {
```

Comment:
again there is no need for an if. also, the logger never prints `ret`. why not do it like that?
```
boolean ret = ...;
LOGGER.info("isLoggedIn: {}", ret);
return ret;
```